src/sentry/api/endpoints/monitor_checkins.py

```diff
@@ -137,9 +146,26 @@ def post(
         result = serializer.validated_data
 
         with transaction.atomic():
+            environment_name = result.get("environment")
+            if not environment_name:
+                environment_name = "production"
```

Comment:
nice

Reply:
oh actually, I wonder if we can give a default in the validator? We can clean that up later